transit_odp/timetables/extract.py

```diff
@@ -219,12 +261,22 @@ def extract_vehicle_journeys(self):
         standard_vehicle_journeys = self.doc.get_all_vehicle_journeys(
             "VehicleJourney", allow_none=True
         )
+
+        df_vehicle_journeys = vehicle_journeys_to_dataframe(standard_vehicle_journeys)
```

Comment:
These changes should be part of dqs changes

Reply:
will remove these changed